### ISSUE TITLE
"Show password" button present in password edit field is not accessible with keyboard:A11y_ASP.NET_Razor Pages_Login_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/aspnetcore#56315]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
## GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_ASP.NET_Web_Jun2024; #.NETCore; #WCAG2.1.1; #DesktopWeb; #Win11; #FTP; #ChromiumEdge; #Keyboard;
## Environment Details:
App name: ASP.NET
Window Version: Win 11
OS Build: 22631.3737
Chromium Edge Version 126.0.2592.61 (Official build) (64-bit)
## Repro Steps:
1. Download the "RazorPagesApp" zip file and run .exe file.
2. Command prompt will open, TAB to "http://localhost:5000" link and press CTRL + ENTER key.
3. TAB to "Login" tab and hit ENTER key. 
4. TAB till password field and give some input "show password" button will appear.
5. TAB till show password button.
6. Observe that "show password" button present in password edit field is not accessible with keyboard.
## Actual Result:
"Show password" button present in password edit field is not accessible with keyboard.
**Note:** Issue also seen with password and confirm password edit field present under register.

**Simila issue observed in Razor Pages Unchanged app**
## Expected Result:
"Show password" button present in password edit field should be accessible with keyboard.
## User Impact:
Keyboard only user will not be able to interact with "Show password" button
## Attachment
https://github.com/dotnet/aspnetcore/assets/124254981/d7ddedc0-ec05-4228-9beb-dd4c4f2da4e2
Simila issue observed in Razor Pages Unchanged app

https://github.com/dotnet/aspnetcore/assets/124254981/0c13f991-23ac-4de3-be0f-2827a2dbe8cc

**Comment by mkArtakMSFT** [comment]
The `Show Password` icon has nothing to do with the framework. We don't control it and it's something provided by the browsers natively. So if you want, you can file an issue against the browsers.
Closing this, as we have no control over it.

/cc @merriemcgaw FYI

**Comment by merriemcgaw** [comment]
@mkArtakMSFT even though I know this is not ours, we need to take it to Office Hours to get it closed and tagged appropriately for internal tracking.

**Comment by kapilvaishna** [comment]
Discussed with Mohammed Ahmed sine it's a by design. so, closing this bug.

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by Raghu-addepalli** [comment]
GithubTags:#A11yBydesign;
